packages/flutter_tools/test/integration.shard/android_plugin_example_app_build_test.dart

```diff
@@ -83,11 +83,6 @@ void main() {
             .join(exampleAppDir.path, 'android', 'gradle', 'wrapper'))
         .deleteSync(recursive: true);
 
-    // Use AGP 3.3.0
-    newBuildGradle = buildGradle.replaceAll(
-        androidPluginRegExp, 'com.android.tools.build:gradle:3.3.0');
-    buildGradleFile.writeAsStringSync(newBuildGradle);
-
```

Comment:
@blasten This version of AGP is not compatible with the latest kotlin, which androidx requires.